Source/IGListAdapter.m

```diff
@@ -308,6 +308,16 @@ - (NSUInteger)sectionForSectionController:(IGListSectionController <IGListSectio
     return [self.sectionMap sectionControllerForObject:object];
 }
 
+- (id)objectForSectionController:(IGListSectionController <IGListSectionType> *)sectionController {
+    IGAssertMainThread();
+    IGParameterAssert(sectionController != nil);
+    
+    NSUInteger section = [self.sectionMap sectionForSectionController:sectionController];
+    IGAssert(section != NSNotFound, @"Section nil for section controller %@", sectionController);
```

Comment:
ahhh I see your commit on the SectionMap! I didn't realize my earlier rebase to my fork didn't quite work as expected; I'll update this real quick.